templates/components/password.build.hbs

```diff
@@ -1,6 +1,9 @@
 <input
   type="password"
   class="form-control"
+  :class="{ disabled: autogenerate }"
```

Comment:
This is a minor point and I don't mind having the `disabled` class here, but I thought having `readonly` means it won't be interactable anyway. Is there any reason this class is necessary?

Reply:
@tonytlwu adding the class seemed to make the input background grey, so it's more obvious about it being readonly.

Reply:
CSS should be using `[readonly]` to color the background grey. Let's leave the `disabled` class in for this purpose. Will update the styling selectors in the future so we're not relying on the wrong class to accomplish this.